CHANGES.rst

```diff
@@ -126,8 +126,8 @@ Bug Fixes
   - Fixed a bug in the ``EPSFStar`` ``register_epsf`` and
     ``compute_residual_image`` computations. [#885]
 
-  - A ValueError is raised if ``aperture_radius`` is not input and
-    cannot be determined from the input ``psf_model``. [#903]
+  - Added fallback ``aperture_radius`` for PSF models without a FWHM or
+    sigma attribute, raising a warning. [#903, #740]
```

Comment:
0.7 was released yesterday.  The new changelog entry should go in 0.8 API changes.

Reply:
Done; I thought this would be an issue but I put the entry where it was supposed to go at the time. It's now in the 0.8 entry with the new release, and should hopefully now reflect an _update_ to 903 instead of replacing, but let me know if the wording isn't ok.